modern/src/main/StatusCard.js

```diff
@@ -129,82 +130,97 @@ const StatusCard = ({ deviceId, onClose }) => {
   return (
     <>
       {device && (
-        <Card elevation={3} className={classes.card}>
-          {deviceImage ? (
-            <CardMedia
-              className={classes.media}
-              image={`/api/media/${device.uniqueId}/${deviceImage}`}
-            >
-              <IconButton size="small" onClick={onClose}>
-                <CloseIcon fontSize="small" className={classes.mediaButton} />
+        <Draggable>
+          <Card elevation={3} className={classes.card}>
+            {deviceImage ? (
+              <CardMedia
+                className={classes.media}
+                image={`/api/media/${device.uniqueId}/${deviceImage}`}
+              >
+                <IconButton
+                  size="small"
+                  onClick={onClose}
+                  onTouchStart={onClose}
+                >
+                  <CloseIcon fontSize="small" className={classes.mediaButton} />
+                </IconButton>
+              </CardMedia>
+            ) : (
+              <div className={classes.header}>
+                <Typography variant="body2" color="textSecondary">
+                  {device.name}
+                </Typography>
+                <IconButton
+                  size="small"
+                  onClick={onClose}
+                  onTouchStart={onClose}
+                >
+                  <CloseIcon fontSize="small" />
+                </IconButton>
+              </div>
+            )}
+            {position && (
+              <CardContent className={classes.content}>
+                <Table size="small" classes={{ root: classes.table }}>
+                  <TableBody>
+                    {positionItems.filter((key) => position.hasOwnProperty(key) || position.attributes.hasOwnProperty(key)).map((key) => (
+                      <StatusRow
+                        key={key}
+                        name={positionAttributes[key].name}
+                        content={(
+                          <PositionValue
+                            position={position}
+                            property={position.hasOwnProperty(key) ? key : null}
+                            attribute={position.hasOwnProperty(key) ? null : key}
+                          />
+                        )}
+                      />
+                    ))}
+                  </TableBody>
+                </Table>
+              </CardContent>
+            )}
+            <CardActions classes={{ root: classes.actions }} disableSpacing>
+              <IconButton
+                color="secondary"
+                onClick={(e) => setAnchorEl(e.currentTarget)}
+                onTouchStart={(e) => setAnchorEl(e.currentTarget)}
```

Comment:
Do we need both `onClick` and `onTouchStart`?

Reply:
Yes, that's the workaround found by @BIhab that makes it work on mobile devices.

Reply:
Yeah, but wouldn't it cause double-calls in some cases? It's also mentioned in that ticket.

Reply:
Where did you read about double calls? What I've found in that ticket is somebody talking about the [double click event](https://github.com/react-grid-layout/react-draggable/issues/597) not being fired on touch devices. There is [another](https://github.com/react-grid-layout/react-draggable/issues/597#issuecomment-1029947048) solution proprosed that adds only an handler to react-draggable. It doesn't duplicate code, but feels even more hacky. What do you prefer? 

```
const handleOnDragStop = e => {
    const { target, type } = e;
    if (type === 'touchend') target.click();
    return true;
  };

...

<Draggable
    ...
    onStop={handleOnDragStop}
    ...
>
        ...
</Draggable>
```

Reply:
I saw this in one of the snippets, so I'm assuming it's to avoid duplicate calls:
```
  const eventHandler = cb => {
    return event => {
      if (isMobile && event.type === 'touchstart') {
        cb(event);
      } else if (!isMobile && event.type === 'click') {
        cb(event);
      }
    };
  };
```

Reply:
> I saw this in one of the snippets, so I'm assuming it's to avoid duplicate calls:
> ```
>   const eventHandler = cb => {
>     return event => {
>       if (isMobile && event.type === 'touchstart') {
>         cb(event);
>       } else if (!isMobile && event.type === 'click') {
>         cb(event);
>       }
>     };
>   };
> ```

Assuming that the called function is the same, why not using an || instead of else if? 
``` 
if ( (isMobile && event.type === 'touchstart') || (!isMobile && event.type === 'click') ) {
          cb(event);
}
```

Reply:
My point is mostly about safety of not doing the check, like it's implemented currently.

Reply:
By the way, what about clicks on the attributes? For example, does "Show Address" link work? Or accumulators.

Reply:
Yes, these links work as well.

Reply:
How come those links work without a need for `onTouchStart`?

Reply:
I'm sorry, they work only on the desktop just like the others.

Reply:
OK, then we have a big issue. I suggest we wait for a proper fix or find some other solution for dragging.

Reply:
For the address filed the `onTouchStart` hack works too, but for the accumulator I think it will get tricky. Guess it's better to hold on as you suggest.

Reply:
What if we set a "handle", so you can only drag by the header? Would it solve the problem with clickable content?

Reply:
We would still need to fix the close button. But that's a good idea, I'll try it.

Reply:
It worked. In fact the header handle is even more intuitive.